utils/scanpypi

```diff
@@ -272,31 +278,69 @@ class BuildrootPackage():
             folder=tmp_pkg,
             name=pkg_filename)
 
+    def find_metadata(self, setup, extract_path):
+        if self.metadata_name in self.setup_args:
+            # setup was called on import, all good
+            return self.setup_args[self.metadata_name]
+        elif hasattr(setup, 'main') and inspect.isfunction(setup.main):
+            # maybe there’s a main function?
+            setup.main()
+        elif hasattr(setup, 'setup_args') and isinstance(setup.setup_args, tuple):
+            setup.setup(*setup_args)
+        elif hasattr(setup, 'setup_args') and isinstance(setup.setup_args, dict):
+            setup.setup(**setup.setup_args)
+        else:
+            # Maybe it’s in the config file
+            config = configparser.ConfigParser()
+            config.read(os.path.join(extract_path, 'setup.cfg'))
+            self.setup_args[self.metadata_name] = {}
+            if 'metadata' in config.sections():
+                self.setup_args[self.metadata_name].update(config['metadata'])
+            if 'options' in config.sections():
+                self.setup_args[self.metadata_name].update(config['options'])
+                if 'install_requires' in self.setup_args[self.metadata_name]:
+                    self.setup_args[self.metadata_name]['install_requires']\
+                        = [ir for ir in self.setup_args[self.metadata_name]['install_requires'].split('\n')
+                           if ir]
+            if 'method' not in self.setup_args[self.metadata_name]:
+                self.setup_args[self.metadata_name]['method'] = 'bdist_wheel'
+            if self.setup_args[self.metadata_name] == {}:
+                raise KeyError("I give up")
+
+        return self.setup_args[self.metadata_name]
+
     def load_setup(self):
         """
         Loads the corresponding setup and store its metadata
         """
         current_dir = os.getcwd()
         os.chdir(self.tmp_extract)
         sys.path.append(self.tmp_extract)
+        print("FINDING SETUP WITH PATH {}".format(sys.path))
```

Comment:
Looks like a few `print`s were added. Just want to double-check that they were intentionally left in